mfa/binding-ui/README.md

```diff
@@ -0,0 +1,159 @@
+[![Ping Identity](https://www.pingidentity.com/content/dam/picr/nav/Ping-Logo-2.svg)](https://github.com/ForgeRock/ping-android-sdk)
```

Comment:
I am not sure we need to provide this UI module. Specially considering we already provide UI examples in the sample app.

@george-bafaloukas-forgerock Do you have plans to create similar module for iOS?

Reply:
I suggest we include the binding-ui module for now. Developers may find it difficult to test device binding without it, so providing the module would make the process more straightforward.
We have documented that the binding-ui module is optional and suggest to build your own interface.